src/action/start.rs

```diff
@@ -14,20 +15,25 @@ const RCON_PASSWORD_LENGTH: usize = 32;
 
 /// Start lazymc.
 pub fn invoke(matches: &ArgMatches) -> Result<(), ()> {
+    // Parse bind address
+    let bind_addr: SocketAddr = matches.get_one::<String>("bind").unwrap().parse().unwrap();
```

Comment:
Maybe we can keep config: public.address intact, or rename it to config: public.bind, that way you don't have to provide a bind address through the command line.

This public bind address could then be taken from the main configuration. Having `--bind` as extra option is good though.

What do you think?

Reply:
Done as requested, the variable for server route is `[server] name`